src/collection.ts

```diff
@@ -704,9 +727,11 @@ export class Collection<TSchema extends Document = Document> {
 
   // allow an override of the schema.
```

Comment:
also curious why these overloads didn't make it into api.js

Reply:
Which ones? they are present in the types: https://github.com/mongodb-js/nodejs-mongodb-legacy/blob/main/mongodb-legacy.d.ts#L404 (unless I'm missing something)